sig-auth/README.md

```diff
@@ -37,6 +37,39 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-auth)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fauth)
 
+## Subprojects
+
+The following subprojects are owned by sig-auth:
+- **audit-logging**
+  - Owners:
+    - Tim Allclair
+- **authenticators**
+  - Owners:
+    - Mike Danese
+    - Jordan Liggitt
+- **authorizers**
+  - Owners:
+    - Jordan Liggitt
+    - Mike Danese
+    - David Eads
+- **cross-cutting-documentation**
```

Comment:
I may be misremembering, but I think we decided to roll this into other areas as well.